toolchain/check/import_cpp.cpp

```diff
@@ -540,7 +540,49 @@ static auto ImportClassObjectRepr(Context& context,
   static_assert(SemIR::CustomLayoutId::FirstFieldIndex == 2);
 
   // TODO: Import vptr(s).
-  // TODO: Import bases.
+
+  // Import bases.
+  for (const auto& base : clang_def->bases()) {
+    if (base.isVirtual()) {
+      // TODO: Handle virtual bases. We don't actually know where they go in the
```

Comment:
For this TODO (and maybe less so about multiple base classes), is it worth making this a `context.TODO` to flag when this path is hit? i.e., this at least seems off the beaten path, so maybe it's okay turning tests with virtual bases into failures with the TODO.